### ISSUE TITLE
pipecolic acid

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7555]
**Preferred term label:**
plasma pipecolic acid is only modestly elevated and that the elevation of AASA in urine, plasma, and cerebral spinal fluid is the most reliable basis for diagnosis.

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
266100 
PMID: 16491085  -- see supplement

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Elevated circulating alpha-aminoadipic semialdehyde concentration
Def: Increased concentration of alpha-aminoadipic semialdehyde in the blood circulation.
PMID:17560822

**Comment by pnrobinson** [comment]
Synonym Piperideine-6-carboxylic acid (P6C)

**Comment by pnrobinson** [comment]
added
